app/models/organization.rb

```diff
@@ -26,7 +26,12 @@ class Organization < ApplicationRecord
   has_many :invoices, dependent: :destroy, inverse_of: :organization
   has_many :invoice_items, through: :invoices
 
+  accepts_nested_attributes_for :bank_accounts, reject_if: :all_blank, allow_destroy: true
+  accepts_nested_attributes_for :categories, reject_if: :all_blank, allow_destroy: true
+
   validates :name, presence: true
+  validates_associated :bank_accounts
```

Comment:
@dissident I think we don't need it. Association with `accepts_nested_attributes_for` should be validate independently. This method is required to validate association on every object save. Im't pretty sure we don't need it.